account_invoice_import_invoice2data/__openerp__.py

```diff
@@ -4,15 +4,19 @@
 
 {
     'name': 'Account Invoice Import Invoice2data',
-    'version': '8.0.1.0.0',
+    'version': '8.0.2.0.0',
     'category': 'Accounting & Finance',
     'license': 'AGPL-3',
     'summary': 'Import supplier invoices using the invoice2data lib',
     'author': 'Akretion,Odoo Community Association (OCA)',
     'website': 'http://www.akretion.com',
     'depends': ['account_invoice_import'],
     'external_dependencies': {'python': ['invoice2data']},
-    'data': [],
+    'data': [
+        'views/invoice2data_template.xml',
+        'security/res_groups.xml',
+        'security/ir.model.access.csv',
```

Comment:
Usually security is loaded before views, since the groups can be used in the view files and you could get unexisting reference errors otherswise.